Services/CO.CDP.EntityVerification/Services/PponService.cs

```diff
@@ -0,0 +1,11 @@
+namespace CO.CDP.EntityVerification.Services;
+
+public class PponService : IPponService
+{
+    public string GeneratePponId(string scheme, string departmentIdentifier)
```

Comment:
We won't get the department identifier. Can be removed.

Reply:
Done